docs/authoring.md

```diff
@@ -78,40 +76,40 @@ To generate a NuGet package for the component, you can simply right click on the
 </files>
 ```
 
-## Using your authored component
-To use the component in a C# app, the authored component just needs to be added as a project/package reference.
+Your component can then be added as a PackageReference to any consumer. 
 
-For native (C++) apps, there are DLLs needed to host your authored component. When you use the automatic NuGet packaging on-build support (in Visual Studio) to make a nupkg for your runtime component, the DLLs/WinMD are automatically added to your nupkg, before the ```GenerateNuspec``` MSBuild step.
+## For native app consumption
 
-### For native app (C++) consumption
-Install your authored component's package -- this will come with a targets file that automatically adds a reference to the component's WinMD and copies the DLLs necessary for native support.
+If you choose to consume your component through a project reference, then some modifications to the native app's `.vcxproj` file are needed.
+Because dotnet will assume a `TargetFramework` for your app that conflicts with `net5`, we need to specify the `TargetFramwork`, `TargetFrameworkVersion` and `TargetRuntime`. 
+Examples of this are seen in the code snippet below. This is needed for this preview version, as we continue working on proper support.
 
-You'll need to use [C++/WinRT](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/intro-to-using-cpp-with-winrt) to consume your API. So make sure you have C++/WinRT installed, and have added `#include <winrt/MyAuthoredComponent.h>` to the file `pch.h` of the native app.  
+You can then use the Visual Studio UI to add a reference to the C#/WinRT component's `csproj` file, and you also need to add a reference to the WinMD file produced 
+for your component. The WinMD can be found in the output (`bin`) directory and the `Generated Files` directory of the authored component's project.

```

Comment:
_for the component_ to be consistent with the rest of the sentence and because the component might not be the consumers.